docs/quick_start/easy_install.md

```diff
@@ -28,7 +28,7 @@ These requirements support the calculation of plane-wave basis in ABACUS. For LC
 Some of these packages can be installed with popular package management system, such as `apt` and `yum`:
 
 ```bash
-sudo apt update && sudo apt install -y libopenblas-openmp-dev liblapack-dev libscalapack-mpi-dev libelpa-dev libfftw3-dev libcereal-dev libxc-dev g++ make cmake bc git
+sudo apt update && sudo apt install -y libopenblas-openmp-dev liblapack-dev libscalapack-mpi-dev libelpa-dev libfftw3-dev libcereal-dev libxc-dev g++ make cmake bc git rapidjson-dev
```

Comment:
Please address requirement explicitly.

Reply:
This is the support rapidjson library needed to add new json output and can be installed directly from the command line. Do I need to add comments in doc?

Reply:
> This is the support rapidjson library needed to add new json output and can be installed directly from the command line. Do I need to add comments in doc?

Yes. Installing by `apt` is not the only method for the users. In fact most supercomputer users can not use `apt`.